src/applications/vaos/actions/newAppointment.js

```diff
@@ -314,7 +314,11 @@ export function openFacilityPage(page, uiSchema, schema) {
       }
 
       if (parentId && !eligibleFacilities?.length) {
-        recordEligibilityFailure('supported-facilities');
+        recordEligibilityFailure(
+          'supported-facilities',
+          typeOfCareId,
+          parentId,
```

Comment:
I think you also want to add the `parseFakeFHIRId` function here. It's annoying, but we can get rid of them in the future.

Reply:
Done